backend/src/graphql/resolver/TransactionResolver.test.ts

```diff
@@ -81,7 +80,7 @@ describe('send coins', () => {
         await mutate({
           mutation: sendCoins,
           variables: {
-            email: 'wrong@email.com',
+            identifier: 'wrong@email.com',
```

Comment:
the tests should reflect the new capabilities of the identifier and use gradido-id-identifier.
I suggest to predefine uuids for our seeds like peter-lustig: 1111-1111111-1111 or simillar.